components/Buttons/src/MDCFloatingButton+Animation.h

```diff
@@ -0,0 +1,37 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCFloatingButton.h"
+
+@interface MDCFloatingButton (Animation)
+
+/**
+ Causes this button to return to its "normal" size.
```

Comment:
I'm a little wary of documenting the exact value (in case we change it) - how about "scaled to less than 0.1% of the original size"?